src/ray/test/run_core_worker_tests.sh

```diff
@@ -22,7 +22,7 @@ fi
 set -e
 set -x
 
-bazel build "//:core_worker_test" "//:mock_worker"  "//:raylet" "//:raylet_monitor" "//:libray_redis_module.so" "@plasma//:plasma_store_server"
+bazel build -c dbg $RAY_BAZEL_CONFIG "//:core_worker_test" "//:mock_worker"  "//:raylet" "//:raylet_monitor" "//:libray_redis_module.so" "@plasma//:plasma_store_server"
```

Comment:
nit, but would be nice to be consistent about quotations. We have them here but not below.

Also, do debug builds work at all (given the `DEBUG` issue)?

Reply:
Yeah debug builds work, it gives good line numbers and symbols in tsan.

Not sure what you mean with the quotations, here we directly run the command whereas below we use $() to run it and assign it to a variable, so they are different uses I think.

Reply:
Oh, in this specific case I'm not referring to `$` but rather to `"//:core_worker_test"` etc.  
Below I see `//:core_worker_test` which is pretty inconsistent w.r.t. quotes.

Reply:
fixed